app/views/partial/_post.html.erb

```diff
@@ -6,4 +6,7 @@
   <p class="div_footer">
     Comments: <%= post.comments_counter %> , Likes: <%= post.likes_counter %>
   </p>
+  <% if can? :delete, @post %>
+    <%= link_to "Delete Post",class="button", user_post_path(user_id: @post.author.id, id: @post.id), data: { turbo_method: :delete } %>
+  <% end %>
```

Comment:
- [ ] Kindly fix this error when I try to `create a new post` I get this error
![image](https://user-images.githubusercontent.com/24703171/227752103-05ac6508-3ac4-44fc-ba1b-7b9fda944d91.png)
and I wasn't able to test functionality. The error happens because of the `link_to` link for `Delete Post` that you added in the partial view of the post.